WcaOnRails/app/javascript/auto-numeric/index.js

```diff
@@ -1,4 +1,4 @@
-import AutoNumeric from 'autonumeric';
+import AutoNumeric from 'autonumeric/dist/autoNumeric.js';
```

Comment:
It might be nice to add a comment here explaining why we're doing this, because it does look a little weird.